pkgs/tools/networking/netdiscover/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, libpcap, libnet, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "netdiscover";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "netdiscover-scanner";
+    repo = pname;
+    rev = version;
+    sha256 = "13fp9rfr9vh756m5wck76zbcr0296ir52dahzlqdr52ha9vrswbb";
+  };
+
+  buildInputs = [ libpcap libnet autoreconfHook ];
+
+  autoreconfPhase = "./autogen.sh";
```

Comment:
Isn't the hook doing that already?

Reply:
I think the hook runs `autoconf`but that is also what `autogen.sh` does https://github.com/netdiscover-scanner/netdiscover/blob/master/autogen.sh